modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountGroupRelLocalServiceImpl.java

```diff
@@ -157,25 +157,8 @@ public List<AccountGroupRel> getAccountGroupRels(
 
 	@Override
 	public List<AccountGroupRel> getAccountGroupRels(
-		String className, long classPK) {
-
-		return accountGroupRelPersistence.findByC_C(
-			_classNameLocalService.getClassNameId(className), classPK);
-	}
-
-	@Override
-	public List<AccountGroupRel> getAccountGroupRels(
-		String className, long classPK, int start, int end,
-		OrderByComparator<AccountGroupRel> orderByComparator) {
-
-		return accountGroupRelPersistence.findByC_C(
-			_classNameLocalService.getClassNameId(className), classPK, start,
-			end, orderByComparator);
-	}
-
-	@Override
-	public List<AccountGroupRel> getAccountGroupRels(
-		String className, long classPK, String keywords, int start, int end) {
+		Long[] accountEntryIds, String className, long classPK, String keywords,
```

Comment:
is there any reason of Long[] instead of the usual long[] ?

Reply:
I was able to change this to long[]